src/libraries/System.Private.CoreLib/src/System/Buffer.cs

```diff
@@ -440,4 +439,329 @@ private static void _BulkMoveWithWriteBarrier(ref byte destination, ref byte sou
 
 #endif // !MONO
     }
+
+    internal static partial class SpanHelpers
```

Comment:
Should this move back to where it was? It does not use the Block helper structs anymore, so the original reason for why it was moved is not there.